### PR TITLE
Consistent docstring formatting, minor spelling changes to docstrings

### PR DESCRIPTION
**Pull request by PostponingCamel**
My text editor Atom automatically removed superfluous whitespaces.
 - Minor spelling changes to docstrings on line 1226, 1793 in ardentryst.py
 - Super minor spelling change to install.md
 - 90% of the docstrings used triple double quote ("""docstring"""). However, there were a few that didn't and used single double quotes instead. I changed these to conform to the triple double-quote convention.

### CODE REVIEW

File: ardentryst.py [hunk 1]
```diff
@@ -1408,7 +1408,7 @@ def Game_SlotMenu(gameobj = None):
                 screen.blit(line_s, line_r)
                 c += 1
 
-            # ->>> 
+            # ->>>
```
Comment (hhirsch) [inline ardentryst.py:1411]: @PostponingCamel Why is this added?

**Comment by hhirsch** [comment]
Besides the open thread this LGTM

**Review by hhirsch — APPROVED** [review]
Looks good to me.

**Review by elle-trudgett — APPROVED** [review]
